package/network/services/samba36/files/samba.init

```diff
@@ -109,10 +108,12 @@ start_service() {
 	procd_open_instance
 	procd_set_param command /usr/sbin/smbd -F
 	procd_set_param respawn
+	procd_set_param file /var/etc/smb.conf
 	procd_close_instance
```

Comment:
just remove the "interface" from the subject as you add more than 1 reload trigger